backend/protocol_rpc/fastapi_rpc_handler.py

```
@@ -0,0 +1,116 @@
+            )
+
+            return JSONRPCResponse(jsonrpc="2.0", result=result, id=request.id)
+
+        except JSONRPCError as e:
+            print(f"FastAPI RPC Handler JSONRPCError: {e}")
+            return JSONRPCResponse(
+                jsonrpc="2.0",
+                error={"code": e.code, "message": e.message, "data": e.data},
+                id=request.id,
+            )
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)